Library/Formula/watch.rb

```diff
@@ -12,15 +18,47 @@ class Watch < Formula
     sha1 "a7c559378bc74cd30d00f962e63d6ee5c705aea1" => :mountain_lion
   end
 
+  depends_on "gettext" => :build
```

Comment:
Is the Right Thing to put the `depends_on` inside an `if` block contingent on `--with-nls`? And then put a parallel condition in the `test do`?